app/src/main/res/values/strings.xml

```diff
@@ -3,4 +3,7 @@
     <string name="rationale_camera">This app needs access to your camera so you can take pictures.</string>
     <string name="rationale_location_contacts">This app needs access to your location and contacts to know where and who you are.</string>
     <string name="rationale_sms">This app needs access to your sms to read all your great messages.</string>
+    <string name="rationale_ask_again">This app needs to open permissions before using the feature.</string>
```

Comment:
Suggest changing to:
'This app may not work correctly without the requested permissions. Open the app settings screen to modify app permissions'